src/guides/debugging-an-example-smart-contract/index.md

```diff
@@ -154,37 +188,53 @@ We next want to interact with the smart contract to see how it works when workin
 1. In the console where `truffle develop` is running, run the following command:
 
    ```javascript
-   SimpleStorage.deployed().then(function(instance){return instance.get.call();}).then(function(value){return value.toNumber()});
+   SimpleStorage.deployed()
+     .then(function (instance) {
+       return instance.get.call();
+     })
+     .then(function (value) {
+       return value.toNumber();
+     });
    ```
 
    This command looks at the SimpleStorage contract, and then calls the `get()` function as defined inside it. It then returns the output, which is usually rendered as a string, and converts it to a number:
 
    ```javascript
```

Comment:
```suggestion
   ```
```

Reply:
Did you mean to delete only the open tag here? I believe commiting this while leaving the  the closing ``` would turn the rest of the document into a code block.